src/main.ts

```diff
@@ -68,6 +110,9 @@ async function run(): Promise<void> {
         break;
       }
       case 'id_token': {
+        if (idTokenAudience === '') {
```

Comment:
I prefer to let this flow through to the API which will reject it instead of doing client-side validation. The upstream API could change and then the client is too restrictive, which isn't great.